components/main-chef-wrapper/lib/version.go

```diff
@@ -0,0 +1,133 @@
+//
+// Copyright (c) Chef Software, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package lib
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"io/ioutil"
+	"path/filepath"
+	"encoding/json"
+	"github.com/chef/chef-workstation/components/main-chef-wrapper/dist"
+)
+
+
+
+var gemManifestMap map[string]interface{}
+var manifestMap map[string]interface{}
+
+func init() {
+	gemManifestMap = gemManifestHash()
+	manifestMap = manifestHash()
+}
+func Version(){
+	if omnibusInstall() == true {
+		showVersionViaVersionManifest()
+	} else {
+		fmt.Fprintln(os.Stderr, "ERROR:", "Can not find omnibus installation directory for", dist.WorkstationProduct)
```

Comment:
This message won't necessarily mean much to the person using it, who doesn't really have awareness of 'omnibus'.  Instead, maybe "$PRODUCT has not been installed via the platform-specific package provided by $DISTRIBUTOR-NAME. Version information is not available."?